Rakefile

```diff
@@ -31,24 +36,30 @@ task :bootstrap do
   sh "brew install npm"
 
   # Continue with install.
-  Rake::Task["install"].invoke
+  Rake::Task[:install].invoke
 end
 
-desc "Build the website"
+desc "Build the website with development features"
```

Comment:
This description feels inaccurate, as it says `build with development features`, but the task runs a build either with or without development features, depending on how environment variables are set. Maybe change it back to just `Build the website`?